packages/editor/src/store/selectors.js

```diff
@@ -1855,6 +1886,39 @@ export const getInserterItems = createSelector(
 	],
 );
 
+/**
+ * Determines whether there are items to show in the inserter.
+ * @param {Object}  state        Editor state.
+ * @param {?string} rootClientId Optional root client ID of block list.
+ *
+ * @return {boolean} Items that appear in inserter.
+ */
+export const hasInserterItems = createSelector(
+	( state, rootClientId = null ) => {
+		const hasBlockType = some(
+			getBlockTypes(),
+			( blockType ) => canIncludeBlockTypeInInserter( state, blockType, rootClientId )
+		);
+		if ( hasBlockType ) {
+			return true;
+		}
+		const hasReusableBlock = some(
```

Comment:
Why assign this to a variable when it's just immediately returned? It's probably just as readable to do:

```js
return some(
  __experimentalGetReusableBlocks( state ),
  ( block ) => canIncludeReusableBlockInInserter( state, block, rootClientId )
);
```

🤷‍♂️

Reply:
An argument can be made that it's a useful explaining variable, since it's not otherwise immediately apparent the intention of the `some`.

https://refactoring.com/catalog/extractVariable.html